sensitivity-api/src/main/java/com/powsybl/sensitivity/converter/SensitivityComputationResultExporter.java

```diff
@@ -30,7 +30,7 @@
     String getComment();
 
     /**
-     * Export a result of a sensitivity computation
+     * Export a result of a sensitivity computation without contingency
```

Comment:
`CsvSensitivityComputationResultExporter` implements this interface and seems to be able to export a result of a sensitivity computation with contingencies, why is "without contingency" specified here?